packages/@sanity/types/package.json

```diff
@@ -55,6 +55,7 @@
   "devDependencies": {
     "@jest/globals": "^29.7.0",
     "@repo/package.config": "workspace:*",
+    "@sanity/insert-menu": "^1.0.1",
```

Comment:
This is a really good question. Sorry about not making that clear. The reason is that we want to use this menu in the visual editing overlays. 

My original plan was to add this new component to the `sanity` core and then export it from there so it can be used in visual editing, but - @stipsan correct me if I'm wrong here - it would be a bit overkill to introduce `sanity` as a dependency there.

![Screenshot 2024-06-10 at 14 22 02](https://github.com/sanity-io/sanity/assets/4602382/afeec547-c633-4ec4-a17f-b861d9b62a21)

Reply:
Ahh I see!
We can double check with someone from @sanity-io/studio-dx (might be best to ask in the DX channel on slack as well), but my gut feeling tells me that if we want to use it in multiple places and one of those places is the `core` then it should exist first in the core and be exported to other places. But definitely should get some clarification from DX :)

Reply:
I think the studio-internal is probably a good place but either that one or just studio works :)

Reply:
I've asked for a second opinion now. Thank you for being thorough 😊

Reply:
Hi @RitaDias, just wanted to add that for the way we want to use this menu in visual editing overlays it's not viable for them to only be available in `sanity` exports, as `sanity` is too large and bundlers other than our own vite setup when you `sanity dev`, really struggle with it.
[Here's one case where we investigated slow builds](https://sanity-io.slack.com/archives/CET4EQG6A/p1715702397597709).
We considered having the `@sanity/insert-menu` on this monorepo, and handle it similar to `@sanity/portable-text-editor`, instead of the `visual-editing` monorepo. But since this monorepo has a publishing flow where all workspace packages are on the same `v3` major it becomes less than ideal for us 😅 
It's also important for us to test `@sanity/insert-menu` on a wide range of frameworks and tools:
- next pages router
- next app router with "use client"
- next app router with RSC
- astro
- sveltekit
- remix
- hydrogen
- nuxt
And since the `visual-editing` monorepo is already setup with most of these already it makes more sense to us :)

Reply:
Rune and Espen have now approved of the use of a new package for this 😇 

https://sanity-io.slack.com/archives/C01B86SLDNX/p1718124803545439?thread_ts=1718023371.382639&cid=C01B86SLDNX

Reply:
👋 
I read that he also had some feedback on the PR itself, was that also updated in the commits that have been pushed since? :)

Reply:
No, nothing has changed since you last look at this PR, except I added a few more placeholder block types to the page building studio 😊

Reply:
I'm assuming those changes will still come on this PR then? :)

Reply:
I had the same question, so I've now asked it in Slack 🙂